docs/control-structures.rst

```diff
@@ -478,6 +478,7 @@ In any case, you will get a warning about the outer variable being shadowed.
     }
 
 
+.. index:: ! safe math, safemath, checked, unchecked
```

Comment:
- I think that `safe math` is superfluous because it will be in the index next to `safe math` anyway.
- This section is the main documentation piece for all of them so according to docs for [.. index::](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-index) you should add exclamation marks in all cases.
- Might be a good idea to add "overflow" and "underflow" too.

```suggestion
.. index:: ! safe math, ! checked, ! unchecked, overflow, underflow
```